packages/ui-extensions/docs/surfaces/point-of-sale/reference/targets/pos.purchase-complete.observe.doc.ts

```diff
@@ -0,0 +1,14 @@
+import type {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+import {ExtensionTargetType} from '../types/ExtensionTargetType';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: ExtensionTargetType.PosCustomerDetailsActionMenuItemRender,
```

Comment:
I think we need to add a new type to `ExtensionTargetType`? Also the file name should be pos.transaction-completed.observe?

Reply:
Yes, thank you